.github/workflows/cicd-2-publish.yaml

```diff
@@ -1,16 +1,17 @@
 name: "CI/CD publish"
 
 on:
-  pull_request:
-    types: [closed]
-    branches:
-      - main
+  push:
```

Comment:
I've changed this publish workflow to run on push to main due to the semantic versioning needing to run on the main branch after a merge. I left the `pull_request` section commented out because I think it still runs on the branch of the PR.

Reply:
I understand the first bit about running on a main push event, but don't follow the second bit. Can you explain a bit more please?

Reply:
@fstewart-nhs the `semantic-release` action won't do anything unless it's on the main or a feature branch. I commented out that `pull_request` line because I wasn't sure if we wanted other actions of this workflow to run on Pull Requests branches, and accept that the `semantic_release` would just be skipped. 

So I suppose my question should have been do we want it included or not?